templates/sg_rule_analysis.yaml

```diff
@@ -239,44 +295,170 @@ Resources:
                                 "BackoffRate": 2
                               }
                             ],
-                            "Next": "QueryVPCFlowLogsSaveToS3",
+                            "Next": "QueryParameters",
                             "ResultPath": null
                           },
-                          "QueryVPCFlowLogsSaveToS3": {
+                          "QueryParameters": {
+                            "Type": "Pass",
+                            "Parameters": {
+                              "queryParams": {
+                                "queryString": "SELECT count(\"interface_id\") as flow_count, interface_id, protocol, flow_direction, if(\"flow_direction\"='ingress',\"srcaddr\",\"dstaddr\") as addr, dstport FROM \"vpcflowlogsathenadatabase\".\"{}\" WHERE dstport is not null and day='{}' and action='ACCEPT' group by interface_id, protocol, flow_direction, if(\"flow_direction\"='ingress',\"srcaddr\",\"dstaddr\"), dstport, dstport having count(interface_id) > 5  order by interface_id, flow_count desc offset {} limit {}",
+                                "outputLocation.$": "States.Format('{}{}', 's3://${AthenaBucket}/', $.path)",
```

Comment:
This is why the csv pull is failing in the flow log parser. Need to correct this output to use yyyy/mm/dd format at end of path e.g. 's3://{$AthenaBucket}/yyyy/mm/dd/

Reply:
see #82